setup.py

```diff
@@ -11,29 +11,31 @@
 
 requirements = [
     'Click>=6.0',
-    # TODO: put package requirements here
+    'bigchaindb-common>=0.0.3',
+    'graphviz'
+]
+
+dev_requirements = [
+    'ipdb',
 ]
 
 test_requirements = [
-    # TODO: put package test requirements here
 ]
 
 setup(
     name='bdb_transaction_cli',
     version='0.1.0',
-    description="A minimal command line tool for create and sign BigchainDB transactions.",
+    description=("A minimal command line tool for create and sign "
+                 "BigchainDB transactions."),
     long_description=readme + '\n\n' + history,
     author="BigchainDB GmbH",
     author_email='dev@bigchaindb.com',
-    url='https://github.com/wrigley/bdb_transaction_cli',
-    packages=[
-        'bdb_transaction_cli',
-    ],
-    package_dir={'bdb_transaction_cli':
-                 'bdb_transaction_cli'},
+    url='https://github.com/bigchaindb/bdb-transaction-cli',
+    packages=['bdb_transaction_cli'],
+    package_dir={'bdb_transaction_cli': 'bdb_transaction_cli'},
     entry_points={
         'console_scripts': [
-            'bdb_transaction_cli=bdb_transaction_cli.cli:main'
+            'bdb=bdb_transaction_cli.__main__:main'
```

Comment:
Yep, well, `python setup.py install` places the bdb exetutable so I can only assume that's what it's doing.